reaper/values.yaml

```diff
@@ -0,0 +1,18 @@
+outputTopicPrefix: raw
```

Comment:
Did you guys do the work of setting sane defaults for minikube?

Reply:
We have now